distribution/docs/src/main/resources/_contents/_plugins/metacardingest-network-contents.adoc

```diff
@@ -0,0 +1,75 @@
+==== Metacard Ingest Network Plugin
+
+Allows the conditional insertion of new attributes on metacards during ingest based on network information from the ingest request; including IP address and hostname.
+
+For the extent of this section, a 'rule' will refer to a configured, single instance of this plugin.
+
+===== Installing the Metacard Ingest Network Plugin
+
+The Metacard Ingest Network Plugin is installed by default during a standard installation in the catalog application.
+
+===== Configuring the Metacard Ingest Network Plugin
+
+Multiple instances of the plugin can be configured by clicking on its configuration title within the configuration tab of the catalog app. Each instance represents a conditional statement, or a 'rule', that gets evaluated for each ingest request. For any request that meets the configured criteria of a rule, that rule will attempt to transform its list of key-value pairs to become new attributes on all metacards in that request.
+
+The condition piece of the rule is divided into two fields: "Criteria" and "Expected Value". The "Criteria" field features a drop-down list containing the four elements for which equality can be tested:
+
+* IP Address
+* Host Name
+* Scheme
+* Context Path
+
+In order for a rule to evaluate to true and the attributes be applied, the value in the "Expected Value" field must be an exact match to the actual value of the selected criteria. For example, if the selected criteria is "IP Address" with an expected value of "192.168.0.1", the rule only evalutes to true for ingest requests coming from "192.168.0.1" and nowhere else.
+
+.Check for IPv6
+IMPORTANT: Verify your system's IP configuration. Rules using "IP Address" may need to be written in IPv6 format.
+
+The key-value pairs within each rule should take the following form: "key = value" where the "key" is the name of the attribute and the "value" is the value assigned to that attribute. Whitespace is ignored unless it is within the key or value. Multi-valued attributes can be expressed in comma-separated format if necessary.
```

Comment:
While it may not be intuitive, `whitespace` can be used as a plural form. It can refer to a single symbol or a collection (such as outer space is the collective sum of the space outside earth, not a single portion).